packages/showcase/src/app/components/tooltip/tooltip.component.scss

```diff
@@ -0,0 +1,5 @@
+.tooltip-for-showcase-container {
```

Comment:
It would be nice to update the tooltip implementation to use the right token for the elevation:
https://www.figma.com/design/O05P2SkMK7aydI2Lxyu7Wi/Mercury-Components?node-id=3007-3844&m=dev